tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -770,11 +769,12 @@ void DxcContext::Recompile(IDxcBlob *pSource, IDxcLibrary *pLibrary,
       NewDefines.size(), pIncludeHandler, &pResult));
   }
 
+#ifndef _WIN32
+  // FIXME: fix crash on linux when not detach pCompileSource.
```

Comment:
Are you sure this crash still happens? It looks like a number of fixes have gone into CComBSTR since this comment was written. Might those have fixed it?

Reply:
It still happens.